vars/setupPMMClient.groovy

```diff
@@ -56,7 +56,8 @@ def call(String SERVER_IP, String CLIENT_VERSION, String PMM_VERSION, String ENA
                     wget -O pmm2-client.tar.gz --progress=dot:giga "https://www.percona.com/downloads/pmm2/${CLIENT_VERSION}/binary/tarball/pmm2-client-${CLIENT_VERSION}.tar.gz"
                 fi
 
-                export BUILD_ID=dont-kill-the-instance
+                export BUILD_ID=dont-kill-the-process
+                export ENKINS_NODE_COOKIE=dont-kill-the-process
```

Comment:
JENKINS

Reply:
we had something like this
export BUILD_ID=dear-jenkins-please-dont-kill
export JENKINS_NODE_COOKIE=dear-jenkins-please-dont-kill
export JENKINS_SERVER_COOKIE=dear-jenkins-please-dont-kill

Reply:
JENKINS_SERVER_COOKIE is useless as per https://stackoverflow.com/questions/39169457/how-to-tell-jenkins-not-to-kill-processes-after-successful-execution-of-job-in-m

Reply:
> and we had `JENKINS_NODE_COOKIE=dontKillMe` before nohup starting pmm-agent, like this: https://github.com/Percona-Lab/jenkins-pipelines/blob/DISTPG-378/vars/setupPMMClient.groovy#L94C21-L94C51

This was not working anyway, since we didn't use the `-E` parameter, that passes the envvars to shell execution. I don't think it is necessary.

Reply:
I tested the fix - it worked :)